packages/@okta/vuepress-theme-prose/util/coveo/listeners.js

```diff
@@ -0,0 +1,70 @@
+// Prevent query suggestions from being displayed until text is entered and
+// there are suggestions available.
+const _registerOmniboxListener = () => {
```

Comment:
My guess is that it's OK, but just making sure we are doing babel translation for arrow fns in this project.

Reply:
Arrows are widely used throughout the project. I think vuepress' build script has babel support out of the box.